apps/shipments/models/shipment.py

```diff
@@ -228,6 +228,7 @@ class ExceptionType(Enum):
 class Shipment(AnonymousHistoricalMixin, models.Model):
     id = models.CharField(primary_key=True, default=random_id, max_length=36)
     owner_id = models.CharField(null=False, max_length=36)
+    assignee_id = models.UUIDField(null=True)
```

Comment:
Interesting - UUIDField looks great. I'm not sure if there was a reason why we didn't use it in the past, or if we just didn't know about it. Good find either way. @mlclay thoughts? Looks like it defaults to uuid4 format.